future work tilføjelser (#71)
SW705E19/report: sections/discussion/future-work.tex

There are 5 tasks that we would like to implement.

\begin{itemize}
    \item Update the select multiple button

>     \item Make tutorinfo inactive

Reviewer: ```suggestion
    \item Make tutor info inactive
```

    \item Extend the recommender to have content based and collaborative approach.
\end{itemize}
\noindent
\textit{Update the select multiple button}:

> During the \autoref{sec:usability-test-section} we found that our current select multiple were unintuitive. 

Reviewer: ```suggestion
During the \autoref{sec:usability-test-section} we found that our current style of selecting multiple options was unintuitive. 
```

> They didn't seem to understand that you were able to select multiple languages, as it only highlighted the selected items with a grey color.

Reviewer: ```suggestion
They did not seem to understand that they were able to select multiple languages, as it only highlighted the selected items with a grey color.
```

> These select multiple buttons should be edited, so that it becomes more intuitive.

Reviewer: ```suggestion
These select multiple buttons should be edited, so that it becomes more intuitive, or a more clear description should be written prior to selecting, properly indicating that it is possible to select multiple options.
```

\\\\
\textit{Make tutor info inactive}:

> If a tutor gets taken his tutor role away, he will still appear on the show services page. 

Reviewer: ```suggestion
If a tutor gets their tutor role removed, they will still appear on the show services page. 
```

> This is because that we do not delete tutor information or his created services.

Reviewer: ```suggestion
This is because we do not delete tutor information or their created services.
```

> This could be fixed by adding an additional attribute to the tutorinfo table in the database to see if they're active.

Reviewer: ```suggestion
This could be fixed by adding an additional attribute to the tutor info table in the database to see if they are active.
```

> If active is set to false, we should not fetch the tutorinfo or services.

Reviewer: ```suggestion
If active is set to false, we should not fetch the tutor info or services.
```

\\\\
\textit{Applying to become a tutor}

> Currently it is not possible to apply for becoming a tutor.

Reviewer: ```suggestion
Currently it is not possible to apply to become a tutor.
```

> This should of course be changed, so that users have the opportunity to apply for that. 

Reviewer: ```suggestion
This should of course be changed, such that users have the opportunity to apply for that. 
```
